<ui version="4.0" >
 <class>Form_usbtray</class>
 <widget class="QWidget" name="Form_usbtray" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>USB Device Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/usbtray_trayicon.png</normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/usbtray_trayicon.png</iconset>
  </property>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/kademarcenter/img/bg/usbtray.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="boto_desconnecta" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>300</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/usbtray_desconecta.png</normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/usbtray_desconecta.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="boto_sortir" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>300</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/desconfigura.png</normaloff>../../../../../usr/share/kademar/utils/kademarcenter/img/desconfigura.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Select the USB device you want to disconnect
 or extract and click on the button "Disconnect"</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget" >
   <property name="geometry" >
    <rect>
     <x>55</x>
     <y>72</y>
     <width>301</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth" >
    <number>1</number>
   </property>
   <property name="horizontalScrollBarPolicy" >
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>16</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/kademarcenter/img/usb.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
